### PR TITLE
An Initial Start to Refactoring (and Standardising Coding Methods)

### PR DESCRIPTION
**Pull request by powertashton**
This PR consists of some refactoring of tables, forms, and processes, alongside medium sized chunks of code being moved around with the intent of moving from the current JS/JQuery/AJAX functions hidden away in multiple different files into using OO'd PHP inline.

More changes may be made either pushed to this PR or in a future PR however I'm submitting this PR now so as to make it slightly less large of a chunk of code to review (Sorry Sandra!)

### CODE REVIEW

**Comment by powertashton** [comment]
Of course it is only just now I see Jim's existing PR.... I suspect our PRs would clash as they're doing the same thing in different ways, happy with whoever's code gets to win though. (And I might steal some code as I continue to finish my own rendition)

**Comment by SKuipers** [comment]
Oh no! There was already a PR? We must have missed that one 😬  Non-core PRs don't show up in the same list as the core, so sometimes they can get a little lost. Oops! Since you're right, we can't accept both PRs at the same time, and you're working on yours more recently, I'll aim to review and merge this PR, then can you see if you can harvest as much as you can from the other PR and add it to your next PR, that way we get the best of both worlds 😅

File: House Points/award.php [hunk 1]
```diff
@@ -1,26 +1,113 @@
...
-        var modpath = '<?php echo $modpath ?>';
-    </script>
-    <?php
-    
-    $pt = new pt($guid, $connection2);
-    $pt->modpath = $modpath;
-    
-    $pt->mainform();
+    echo "<p>&nbsp;</p>";
+        echo "<h3>Award house points to house</h3>";
```
Comment (SKuipers) [inline House Points/award.php:32]: This can be done with the $form->setTitle() method, for consistency.

File: House Points/award.php [hunk 2]
```diff
@@ -1,26 +1,113 @@
...
+         $row = $form->addRow();
+            $row->addFooter();
+            $row->addSubmit();
+
+        echo $form->getOutput();
+
+        echo "<div>&nbsp;</div>";
+        echo "<p id='msg' style='color:blue;'></p>";
+        
+        //TODO: rewrite the code to maybe use a chained or something rather than this weird ajax...
```
Comment (SKuipers) [inline House Points/award.php:77]: Haha yeah 😅

**Review by SKuipers — APPROVED** [review]
Looking good, this is no easy refactor, it's practically a rewrite, thanks for taking this on! My apologies for mixup with the existing PR 😬 I'm going to accept this and begin testing it out, then we can look to release it as a new version once the refactoring is done. Thanks! 👍

**Pull request merged**